src/ring-call-channel.c

```diff
@@ -633,6 +651,13 @@ ring_call_channel_fill_immutable_properties(TpBaseChannel *base,
     tp_dbus_properties_mixin_fill_properties_hash (obj, props,
       TP_IFACE_CHANNEL_INTERFACE_SERVICE_POINT, "InitialServicePoint",
       NULL);
+
+  if (!RING_STR_EMPTY(self->priv->imsi)) {
+    GValue* imsi = g_slice_new0 (GValue);
+    g_value_init (imsi, G_TYPE_STRING);
+    g_value_set_string (imsi, self->priv->imsi);
```

Comment:
It seems that the above three lines can be replaced with `tp_g_value_slice_new_string(self->priv->imsi)`